html/changelogs/geeves-artifact_ranges.yml

```diff
@@ -0,0 +1,6 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - bugfix: "Reduced long-range artifact effect max range from 200 tiles, to 20."
```

Comment:
Not a bugfix, because there is no bug to fix here.

Reply:
I don't agree, but alright.